pkgs/development/tools/language-servers/nixd/default.nix

```diff
@@ -16,22 +16,24 @@
   nlohmann_json,
   pkg-config,
   testers,
+  python312,
 }:
 
 let
   common = rec {
-    version = "2.2.3";
+    version = "2.3.0";
 
     src = fetchFromGitHub {
       owner = "nix-community";
       repo = "nixd";
       rev = version;
-      hash = "sha256-1i4U012vqw84NuD5cjgfVX/he4vW9MyEeIetJ55g280=";
+      hash = "sha256-iB6Hc3kgtsZ360JlAhPqCRWFTf3tawHEKYAeYt7DM4E=";
     };
 
     nativeBuildInputs = [
       meson
       ninja
+      python312
```

Comment:
Wouldn't it be better to use `python3` instead?

Reply:
It really does, I just pushed the changes. Thanks!